### PR TITLE
[MOS-288] Split default and fallback fonts to separate entries

### PR DESCRIPTION
**Pull request by paweljonskim**
And fix emojis display

### CODE REVIEW

**Review by mjan-dev — APPROVED** [review]

**Review by niepiekm — APPROVED** [review]

File: doc/fonts.md [hunk 1]
```diff
@@ -19,18 +19,20 @@ Short example:
...
     "info" : {
-        "default_font_name" :       "dejavu_sans_bold_27",
-        "default_font_type_name" :  "dejavu_sans"
+        "fallback_font" : "dejavu_sans_bold_27",
+        "default_font": "gt_pressura_bold_27",
+        "default_font_family" : "gt_pressura"
     },
     "style": {
-        "mediumboldtitle":      "dejavu_sans/dejavu_sans_bold_27.mpf",
+        "small":      "gt_pressura/gt_pressura_regular_24.mpf",
```
Comment (SP2FET) [inline doc/fonts.md:27]: is the indentation correct?

**Review by SP2FET — APPROVED** [review]

File: doc/fonts.md (hunk 1 above)
Comment (paweljonskim) [reply doc/fonts.md:27]: yem why not? :o

File: module-gui/gui/core/FontManager.cpp [hunk 2]
```diff
@@ -152,11 +153,13 @@ namespace gui
...
-        if (fallback_font != nullptr) {
-            for (auto font : fonts) {
-                font->setFallbackFont(fallback_font);
-            }
+        auto fallback_font = find(fallbackFontName);
+        if (not fallback_font) {
+            return false;
+        }
+
+        for (auto font : fonts) {
```
Comment (swichu91) [inline module-gui/gui/core/FontManager.cpp:161]:
```suggestion
        for (auto& font : fonts) {
```

Comment (swichu91) [reply module-gui/gui/core/FontManager.cpp:161]: Without it, each font will be created and then destroyed instead of just passing reference.

**Review by swichu91 — APPROVED** [review]

**Review by niepiekm — APPROVED** [review]

**Pull request merged**